apps/src/lib/ui/simpleSignUp/workshop/WorkshopWelcomeBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Typography from '@cdo/apps/componentLibrary/typography/Typography';
+import i18n from '@cdo/locale';
+
+import codeLogo from '../assets/codeLogo.svg';
+
+import styles from '../link-account.module.scss';
+
+const WorkshopWelcomeBanner = () => {
+  return (
+    <div className={styles.welcomeContainer}>
+      <div className={styles.welcomeIconContainer}>
+        <img src={codeLogo} alt={i18n.codeLogo()} />
+      </div>
+
+      <div className={styles.titleContainer}>
+        <Typography semanticTag={'h1'} visualAppearance={'heading-xxl'}>
+          {i18n.accountWelcomeBannerHeaderLabel()}
+        </Typography>
```

Comment:
Nit: this can be simplified using `<Heading1>` (you'd need to import it above in the Typography component). Since it's a legit H1 you can remove the `visualAppearance` prop. 
```suggestion
        <Heading1>{i18n.accountWelcomeBannerHeaderLabel()}</Heading1>
```